### ISSUE TITLE
Can't build because of missing GodotSharpTools.dll

### ISSUE THREAD

**Issue by karroffel** [neikeq/GodotSharp#30]
Fresh clone of Godot (6d233c651b21ecaef78fbb20d0365a22919b72b1), fresh clone of GodotSharp (073280a882b8c9db8da3103345cc471069c8f19e)

I cp'd the `mono` directory into `modules/`, it's correctly copied and all.

```
[karroffel@karroffel-pc Monot]$ scons p=x11 use_llvm=yes builtin_openssl=yes mono_glue=no -j 6
scons: Reading SConscript files ...
Enabling ALSA
Enabling PulseAudio
scons: done reading SConscript files.
scons: Building targets ...
[  0%] mono_build_solution(["bin/GodotSharpTools.dll"], ["modules/mono/editor/GodotSharpTools/GodotSharpTools.sln"])
[  2%] Compiling ==> platform/x11/godot_x11.cpp
[  2%] Compiling ==> platform/x11/context_gl_x11.cpp
[  4%] Compiling ==> platform/x11/os_x11.cpp
[  4%] Compiling ==> platform/x11/key_mapping_x11.cpp
[  4%] scons: *** [bin/GodotSharpTools.dll] No such file or directory
Compiling ==> platform/x11/joypad_linux.cpp
scons: building terminated because of errors.
```

**Comment by karroffel** [comment]
Changing the arguments seems to fix that error.

https://github.com/neikeq/GodotSharp/blob/073280a882b8c9db8da3103345cc471069c8f19e/mono/SCsub#L106-L107

**Comment by neikeq** [comment]
Fixed by cad97f9.
